test/e2e/page-objects/pages/network-manager.ts

```diff
@@ -31,7 +31,7 @@ class NetworkManager {
   private readonly networkListItem = (networkName: string) =>
     `[data-testid="network-list-item-${networkName}"]`;
 
-  private readonly tabList = '.tabs__list.network-manager__tab-list';
+  private readonly tabList = '.network-manager__tab-list';
```

Comment:
We shouldn't use atomic UI component selectors for tests. Even using a component's class name is less reliable than setting a test ID at the top level of the component related to the feature.